useCase/useCase.go

```diff
@@ -0,0 +1,40 @@
+package useCase
+
+import (
+	"strconv"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/app"
+	"github.com/Diegoplas/go-bootcamp-deliverable/utils"
+)
+
+func GetPokemonFromCSV(wantedIndex int) utils.PokemonData {
+
+	csvLines := app.CSVReader()
+
+	for _, line := range csvLines {
+
+		pokemonID, _ := strconv.Atoi(line[0])
+
+		if pokemonID == wantedIndex {
+
+			if line[3] == "" {
+				line[3] = " - "
+			}
+
+			legendary := false
+			if line[4] == "TRUE" {
+				legendary = true
```

Comment:
This could be simplified `legendary := line[4] == "TRUE"`.

Also, that TRUE string could be a constant, or even better, there is a function I think on the strings package that can parse a bool string like you have already: https://pkg.go.dev/strconv#ParseBool